src/OpenConext/ProfileBundle/Resources/views/MyServices/AttributeList/idp.html.twig

```diff
@@ -1,10 +1,13 @@
 <p>{{ 'profile.my_services.attribute_release_description'|trans }}</p>
-
+{% set showMotivations = specifiedConsent.hasMotivations %}
```

Comment:
Might be simpler just checking hasMotivations twice instead of assigning variables inside the template, what do you think?